modules/mbedtls/tls_context_mbedtls.cpp

```diff
@@ -32,6 +32,10 @@
 
 #include "core/config/project_settings.h"
 
+#ifdef TOOLS_ENABLED
+#include "editor/editor_settings.h"
+#endif // TOOLS_ENABLED
```

Comment:
Technically not editor code shouldn't include editor headers (see #53295), but we have quite a few exceptions.
In this case I don't see any easy way to avoid this, it would need to be passed as a argument from where mbedtls is used in the editor, and that seems worse than allowing this exception.